doc/dashboards/customize.md

```diff
@@ -0,0 +1,21 @@
+# Customize your dashboard
+
+To deploy a dashboard you need two files:
+
+1. A Jupyter notebook files (`.ipynb`)
+2. A dependencies file (`requirements.txt`)
```

Comment:
Can users provide `environment.yml` as well?

Reply:
no, we currently only support requirements.txt